src/Page.js

```diff
@@ -814,10 +815,11 @@ class Page {
           this.addLayoutFiles();
           this.collectHeadFiles(baseUrl, hostBaseUrl);
 
-          this.content = nunjucks.renderString(this.content, {
-            baseUrl,
-            hostBaseUrl,
-          });
+          this.content = nunjucks.renderString(this.content,
```

Comment:
Not sure if this is personal preference, but I find the original indentation (aligned to the left column) easier to read than aligning with the left opening bracket.